search-commons/src/main/java/no/unit/nva/search/common/AsType.java

```diff
@@ -134,6 +142,6 @@ public String asLowerCase() {
 
     @Override
     public String toString() {
-        return value;
+        return nonNull(value) ? value : EMPTY_STRING;
```

Comment:
Part of the reason I created AsType, was to avoid null object.
Returning "" from toString when it is null or empty, doesn't seem to be a bad idea here....